src/systems/ec/implementations/react/components/table/stories/Index.jsx

```diff
@@ -1,9 +1,33 @@
+/* eslint-disable react/react-in-jsx-scope, import/no-extraneous-dependencies */
```

Comment:
The `import/no-extraneous-dependencies` is 392 occurences in 391 files. We'll need to see the [options](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md#options) and tweak this, it's crazy to carry ignore rules like this along the whole project

Reply:
Created https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-1247 for following up on all the ignore rules updates.

Reply:
ok perfect, I'm leaving them for now.